src/olympia/stats/templates/stats/stats.html

```diff
@@ -89,6 +89,14 @@ <h1>
 {% endblock %}
 
 {% block content %}
+  {% if waffle.switch('disable-bigquery') %}
+    <div class="notification-box info">
+      {% trans %}
+        Sorry, we had to disable the statistics temporarily.
```

Comment:
Feels weird to use "we" on public pages like this. Probably not a huge deal since it's a panic switch though.

Reply:
I'll rephrase a bit